controllers/gatekeeper_controller.go

```diff
@@ -963,6 +971,44 @@ func setNamespaceSelector(
 	return setWebhookConfigurationWithFn(obj, webhookName, setNamespaceSelectorFn)
 }
 
+func setOperators(
+	obj *unstructured.Unstructured, operations *[]v1alpha1.OperationType, webhookName string,
+) error {
+	// If no operations is provided, no override for operations
+	if operations == nil {
+		return nil
+	}
+
+	setOperatorsFn := func(webhook map[string]interface{}) error {
+		rules := webhook["rules"].([]interface{})
+		if rules[0] == nil {
+			return nil
+		}
+
+		converted := make([]interface{}, len(*operations))
```

Comment:
The standard way to do this is `make([]interface{}, 0, len(*operations))` which sets the size to 0 and capacity to the ultimate size. Then you use `append` when adding an item but it won't cause a memory allocation.

This is better because your current way creates a slice with `len(*operations)` of `[]interface{}` objects, so Go has to make memory allocations for all these `[]interface{}` that will be thrown away right after.